### PR TITLE
chore: update plugins team

### PR DESCRIPTION
**Pull request by salmanm**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,6 +234,7 @@ Team members are listed in alphabetical order.
 * [__Vincent Le Goff__](https://github.com/zekth)
 * [__Maksim Sinik__](https://github.com/fox1t), <https://twitter.com/maksimsinik>, <https://www.npmjs.com/~fox1t>
 * [__Manuel Spigolon__](https://github.com/eomm), <https://twitter.com/manueomm>, <https://www.npmjs.com/~eomm>
+* [__Salman Mitha__](https://github.com/salmanm), <https://www.npmjs.com/~salmanm>
```
Comment (jsumners) [inline README.md:237]: This line should be before line 235.

Comment (zekth) [reply README.md]: Alphabetical order

**Review by mcollina — APPROVED** [review]
lgtm

**Review by RafaelGSS — APPROVED** [review]
LGTM.

**Review by zekth — APPROVED** [review]
lgtm. Welcome!

File: README.md (hunk 1 above)
Comment (salmanm) [reply README.md]: Ah, by **last** name. Didn't notice. 🤦
thanks

**Pull request merged**